exporter/kafkaexporter/config_test.go

```diff
@@ -71,5 +71,7 @@ func TestLoadConfig(t *testing.T) {
 				Backoff: defaultMetadataRetryBackoff,
 			},
 		},
+		CompressionCodec: defaultCompression,
+		CompressionLevel: defaultCompressionLevel,
```

Comment:
Maybe it would be good to try some non-default values for these in the [testdata config](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/kafkaexporter/testdata/config.yaml)?

Reply:
Thanks!
> why did you decide to make compression and compression_level separate settings?

The Kafka exporter basically uses sarama as the producer to produce messages to Kafka. So I think it would be the best to follow the sarama's specification about the compression options.

> Maybe it would be good to try some non-default values for these in the testdata config?

Yes, I should add more tests.